maat-court-data-api/src/main/java/gov/uk/courtdata/reporder/controller/RepOrderController.java

```diff
@@ -85,8 +85,20 @@ public ResponseEntity<Object> getRepOrderMvoByRepIdAndVehicleOwner(@PathVariable
 
     }
 
+    @PutMapping
+    @Operation(description = "Update a rep order record")
+    @ApiResponse(responseCode = "200", content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE))
+    @ApiResponse(responseCode = "400", description = "Bad Request.", content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE, schema = @Schema(implementation = ErrorDTO.class)))
+    @ApiResponse(responseCode = "500", description = "Server Error.", content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE, schema = @Schema(implementation = ErrorDTO.class)))
+    public ResponseEntity<Object> updateRepOrder(@Parameter(description = "Update a rep order record", content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE, schema = @Schema(implementation = UpdateRepOrder.class))) @RequestBody UpdateRepOrder updateRepOrder) {
+        log.debug("Update Rep order request received for repId : {}", updateRepOrder.getRepId());
+        maatIdValidator.validate(updateRepOrder.getRepId());
+        repOrderService.updateRepOrder(updateRepOrder);
+        return ResponseEntity.ok().build();
+    }
+
```

Comment:
Does this need to be a separate endpoint, could we not re-purpose the find endpoint and simply add a query string parameter to filter on the sentence order date? That seems much simpler.

I think the best solution here would be to use a `HEAD` request on the same URL for retrieving repOrders that returns the count metadata as a header.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/HEAD